packages/react-search-ui/README.md

```diff
@@ -69,10 +69,218 @@ Connectors are explained in the top level [README](../../README.md). The first
+| view | Component | no        | [PagingInfo](../react-search-ui-views/src/PagingInfo.js) |         | Used to override the default view for this Component. |
+
+Example:
+
+```jsx
+
+import { PagingInfo } from "@elastic/react-search-ui";
+
+...
+
+<PagingInfo />
+```
+
+#### Results<a id="results"></a>
+
+Shows all results. This is a convenience, you could also iterate over the
+results yourself and render each result.
+
+Properties:
+
+| Name         | type      | Required? | Default                                            | Options | Description                                           |
+| ------------ | --------- | --------- | -------------------------------------------------- | ------- | ----------------------------------------------------- |
+| renderResult | Component | no        | [Result](../react-search-ui-views/src/Result.js)   |         | Used to override individual Result views.             |
+| titleField   | String    | no        |                                                    |         | Name of field to use as the title from each result.   |
+| urlField     | String    | no        |                                                    |         | Name of field to use as the href from each result.    |
+| view         | Component | no        | [Results](../react-search-ui-views/src/Results.js) |         | Used to override the default view for this Component. |
+
+Example:
+
+```jsx
+
+import { Results } from "@elastic/react-search-ui";
+
+...
+
+<Results titleField="title" urlField="nps_link" />
+```
+
+#### ResultsPerPage<a id="c_resultsperpage"></a>
+
+Shows a selector for selecting the number of results to show per page. Uses
+20, 40, 60 as options.
+
+Properties:
+
+| Name | type      | Required? | Default                                                          | Options | Description                                           |
+| ---- | --------- | --------- | ---------------------------------------------------------------- | ------- | ----------------------------------------------------- |
+| view | Component | no        | [ResultsPerPage](../react-search-ui-views/src/ResultsPerPage.js) |         | Used to override the default view for this Component. |
+
+Example:
+
+```jsx
+
+import { ResultsPerPage } from "@elastic/react-search-ui";
+
+...
+
+<ResultsPerPage />
+```
+
+#### SearchBox<a id="c_searchbox"></a>
+
+Properties:
+
+| Name       | type      | Required? | Default                                                | Options | Description                                           |
+| ---------- | --------- | --------- | ------------------------------------------------------ | ------- | ----------------------------------------------------- |
+| inputProps | Object    | no        |                                                        |         | Props of 'input' element to be passed through.        |
```

Comment:
I'm confused about what the options are in this case

Reply:
@brianearwood Updated, does that make more sense?

Reply:
Is `placeholder` the only option?

Reply:
It could be any attribute of an input element